src/viewProviders/flinkStatements.test.ts

```diff
@@ -166,7 +230,7 @@ describe("FlinkStatementsViewProvider", () => {
   });
 
   describe("getTreeItem()", () => {
-    it("returns FlinkStatementTreeItem", () => {
+    it("with label set to statement name", () => {
```

Comment:
Minor: should these be combined? `getTreeItem() returns FlinkStatementTreeItem with label set to statement name`?